### ISSUE TITLE
Make a website?

### ISSUE THREAD

**Issue by skyem123** [Nightfall/Nightfall#3]
Well, it sounds like something fun and interesting to do... we should have a website that has information about the stuff here and about the people here.
Maybe we could use the [nightfall.moe](http://nightfall.moe/) domain?

@Victorious3, what do you think of this?

**Comment by RX14** [comment]
Make nightfall.moe be for nightfall, and vic.nightfall.moe be for vic?

**Comment by skyem123** [comment]
yeah.

**Comment by Victorious3** [comment]
Agreed, but I'm way too lazy to do any of this~

**Comment by skyem123** [comment]
But you are the one who controls the domain...

._.

**Comment by RX14** [comment]
No, it's all just http requests to my dedi, I control the virtual hosts.

**Comment by skyem123** [comment]
Oh, that's good! So... how should we make a website?

**Comment by skyem123** [comment]
I have no idea how to generate a static website (Jeykll is for blogs, sadly) for this...

**Comment by RX14** [comment]
no, jekyll isn't just for blogs, the novaapi website is in jekyll.

**Comment by skyem123** [comment]
Oh... interesting...
